ws/templates/help/leaders/ws_gear.html

```diff
@@ -49,87 +49,94 @@ <h4>Special Gear</h4>
   </ul>
 </div>
 
-<h2 id="group">Group Gear</h2>
+<h2 id="group">Group Safety Gear</h2>
 <p>
   Trip leaders must ensure that collectively their group carries the following gear according to trip type.
 </p>
 
 <div class="well" id="a_trip">
   <h4>A Trip</h4>
   <ul>
-    <li>MITOC first-aid kit</li>
-    <li>Extra clothing</li>
-    <li>Extra food</li>
-    <li>Duct tape</li>
-    <li>Parachute cord or webbing</li>
-    <li>Foam pad (optional)</li>
+    <li>One MITOC first aid kit,</li>
+    <li>Extra layers (clothing),</li>
+    <li>Extra food and water,</li>
+    <li>Two foam pads,</li>
+    <li>Duct tape or climbing tape, and</li>
+    <li>Parachute cord or webbing.</li>
```

Comment:
Can you remove the trailing commas + "and" on each of these? HTML lists don't need the punctuation, and there's accessibility reasons & such to not define the text this way.